src/apps/packet_filter/pcap_filter.lua

```diff
@@ -37,13 +41,36 @@ function PcapFilter:new (conf)
       shm = { rxerrors = {counter}, sessions_established = {counter} }
    }
    if conf.state_table then conntrack.define(conf.state_table) end
+
+   alarms.add_to_inventory {
+      [{alarm_type_id='filtered-packets', alarm_type_qualifier=conf.alarm_type_qualifier}] = {
+         resource=tostring(S.getpid()),
+         has_clear=true,
+         description="Total number of filtered packets"
+      }
+   }
+   local alarm_key = {
+      resource=tostring(S.getpid()),
+      alarm_type_id='filtered-packets',
+      alarm_type_qualifier=conf.alarm_type_qualifier
```

Comment:
Is it OK if this is nil?

Reply:
Likely not, better to set to empty string if nil.

Reply:
`alarm-type-quailifier` is not mandatory either in the definition of alarm-types and alarms, so I think it's fine if it's nil.